cmd/drone-server/server.go

```diff
@@ -51,6 +51,12 @@ var flags = []cli.Flag{
 		Usage:  "server address",
 		Value:  ":8000",
 	},
+	cli.StringFlag{
+		EnvVar: "DRONE_HOOK_HOST",
+		Name:   "hook-host",
+		Usage:  "for generating webhook links",
```

Comment:
Maybe something like override hook host? Currently it suggests that this must be set